THScrollViewMinimap/THMinimap/THMinimapView.swift

```diff
@@ -7,45 +7,91 @@
 //
 
 import UIKit
+import AVFoundation
 
 open class THMinimapView: UIView {
+    public var focusedBorderWidth: CGFloat = 2.0
+    public var focusedBorderColor: CGColor = UIColor.yellow.cgColor
+    private var focusedBoxView: UIView = UIView()
+    private var scrollView: UIScrollView = UIScrollView()
+    private var minimapImageView: UIImageView = UIImageView()
+    private var scrollImageView: UIImageView = UIImageView()
+    private var downSizeRatio: CGFloat = 4.0
+    private var image = UIImage()
+    private var minimapImage: UIImage?
+    convenience init(scrollView: UIScrollView, downSizeRatio: CGFloat, image: UIImage) {
+        self.init(frame: CGRect.zero)
+        setUp(scrollView: scrollView, downSizeRatio: downSizeRatio, image: image, minimapImage: nil)
+    }
+    convenience init(scrollView: UIScrollView, downSizeRatio: CGFloat, image: UIImage, minimapImage: UIImage) {
+        self.init(frame: CGRect.zero)
+        setUp(scrollView: scrollView, downSizeRatio: downSizeRatio, image: image, minimapImage: minimapImage)
+    }
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
```

Comment:
Is `THMinimapView`can still be used from storyboard with `setUp(scrollView:downSizeRatio:image:minimapImage)` ?
Call default `setUp()`function, or you should handle required property from another function.(not from init)
For example,

```swift
let minimapView = THMinimapView()
minimapView.setup(scrollView: scrollView, downSizeRatio: downSizeRatio, image: image, minimapImage: minimapImage)
```